ansible-ipi-install/roles/installer/tasks/60_deploy_ocp.yml

```diff
@@ -9,4 +9,42 @@
   shell: |
     /usr/local/bin/openshift-baremetal-install --dir {{ dir }} --log-level debug create cluster
   tags: install
+  when: increase_bootstrap_timeout is not defined and increase_install_timeout is not defined
 
+- name: Deploy OpenShift Cluster with extended timeouts
+  block:
+    - name: Run OpenShift Cluster install as async task
+      shell: |
+        /usr/local/bin/openshift-baremetal-install --dir {{ dir }} --log-level debug create cluster
+      async: 3600
+      poll: 0
+      ignore_errors: yes
+      register: installer_result
+
+    - name: Wait for kubeconfig File
```

Comment:
The right module/better fit would be wait_for https://docs.ansible.com/ansible/latest/modules/wait_for_module.html. Stat always seems to return success.

Reply:
Indeed the stat module doesn't behave the way I expect (IMO it behaves wrongly, but that's for a different forum). Updated now to use wait_for